bessctl/conf/testing/run_module_tests.bess

```diff
@@ -156,15 +153,19 @@ def monitor_task(module, wid):
 def run_tests():
     DID_TESTS_FAIL = False
 
-    path = os.path.dirname(os.path.realpath(sys.argv[0]))
+    # Env variable used to run a subset of tests
+    test_name = $TEST_NAME!'*'
+
+    default_path = os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])),
```

Comment:
It might be good to use `os.path.abspath` rather than `os.path.realpath` in case something changes directories at some point, but this should be fine for now.